src/tests/ftest/cart/util/cart_logparse.py

```diff
@@ -604,7 +604,7 @@ def new_iter(self,
         self._raw = raw
 
         if stateful:
-            if not pid:
+            if pid is None:
```

Comment:
Was it failing before due to getting back a 0?

Reply:
yes, it does. since pid is being changed to rank with the patch.